polkadot/runtime/rococo/src/lib.rs

```diff
@@ -998,6 +998,7 @@ impl coretime::Config for Runtime {
 	type Currency = Balances;
 	//type WeightInfo = weights::runtime_coretime::WeightInfo<Runtime>;
 	type ExternalBrokerOrigin = EnsureXcm<OnlyBroker>;
+	type WeightInfo = runtime_common::coretime::TestWeightInfo;
```

Comment:
We should use the benchmark result here.

Reply:
Wasn't sure if it was okay to add the benchmark to Rococo yet. I'll do that!